rele/client.py

```diff
@@ -59,3 +61,12 @@ def publish(self, topic, data, blocking=False, **attrs):
 
         future.result(timeout=self.PUBLISH_TIMEOUT)
         return future
+
+    def _build_metrics(self, topic):
+        return{
+            'name': 'publications',
+            'data': {
+                'agent': settings.BASE_DIR.split('/')[-1],
```

Comment:
We should really think about making this configurable. We use this elsewhere, but this should not be the default behavior for widespread release.  In a later PR, instead this could be `settings.RELE_PROJECT_NAME`